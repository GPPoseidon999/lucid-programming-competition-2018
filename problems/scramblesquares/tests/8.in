5 5
3 9 1 2
8 2 8 1
0 1 3 7
7 1 3 2
3 8 6 2
2 8 9 2
1 7 2 8
9 4 7 8
1 1 0 2
0 8 0 8
8 7 4 7
2 6 7 3
3 7 4 0
7 2 2 6
6 6 7 1
5 3 4 9
4 2 0 9
4 2 2 4
3 9 9 0
1 0 5 3
2 2 1 6
6 2 8 8
0 3 6 2
7 3 3 9
4 9 3 3
