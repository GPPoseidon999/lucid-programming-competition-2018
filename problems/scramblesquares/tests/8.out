\ 6 /\ 2 /\ 7 /\ 8 /\ 1 /
1 X 66 X 22 X 11 X 22 X 1
/ 7 \/ 1 \/ 3 \/ 8 \/ 0 \
\ 7 /\ 1 /\ 3 /\ 8 /\ 0 /
9 X 33 X 00 X 77 X 77 X 1
/ 3 \/ 5 \/ 4 \/ 4 \/ 3 \
\ 3 /\ 5 /\ 4 /\ 4 /\ 3 /
0 X 99 X 33 X 99 X 22 X 9
/ 9 \/ 4 \/ 3 \/ 0 \/ 1 \
\ 9 /\ 4 /\ 3 /\ 0 /\ 1 /
8 X 44 X 22 X 88 X 88 X 7
/ 7 \/ 2 \/ 6 \/ 0 \/ 2 \
\ 7 /\ 2 /\ 6 /\ 0 /\ 2 /
6 X 22 X 88 X 22 X 33 X 6
/ 2 \/ 9 \/ 8 \/ 6 \/ 7 \
